src/components/CategoryProducts.vue

```diff
@@ -34,6 +34,20 @@ export default class CategoryProducts extends Vue {
     return `${window.location.pathname}/${this.category}`;
   }
 
+  get sortedProducts() {
+    const sortedProducts: Product[] = [];
+    const productNames = this.products.map(product => product.name).sort();
+
+    productNames.forEach(name => {
+      this.products.forEach(product => {
+        if (product.name === name) {
+          sortedProducts.push(product);
+        }
+      });
+    });
+    return sortedProducts;
+  }
```

Comment:
This can be simplified with a [comparator function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#description)
```suggestion
  get sortedProducts() {
    return this.products.sort((product1, product2) => (product1.name < product2.name ? -1 : 1));
  }
```